api/app/controllers/review.controller.ts

```diff
@@ -161,7 +184,7 @@ export async function findOne(req: Request, res: Response) {
           message: `Review with id=${id} does not exist`,
         });
       }
-      res.status(200).send(data);
+      return res.status(200).send(await transcriptWrapper(data.transcript))
```

Comment:
To keep the response of this endpoint the same as before (therefore not breaking FE), you can change this to this:
```suggestion
      const transcript = await transcriptWrapper(data.transcript)
      return res.status(200).send({ ...data.dataValues, transcript })
```